packages/host/app/components/ai-assistant/card-picker/index.gts

```diff
@@ -111,8 +109,8 @@ export default class AiAssistantCardPicker extends Component<Signature> {
       }
       .attach-button:hover:not(:disabled),
       .attach-button:focus:not(:disabled) {
-        --icon-color: var(--boxel-highlight-hover);
-        color: var(--boxel-highlight-hover);
+        --icon-color: var(--boxel-600);
+        color: var(--boxel-600);
         background: none;
         box-shadow: none;
       }
```

Comment:
Maybe we should get rid of this section. The default hover/focus looks much nicer with the updates.

Reply:
Do you mean that it should look like a `text-only` button? Currently it’s a `pill` so removing things makes it look like this:

<img width="352" alt="Boxel 2024-10-28 16-59-34" src="https://github.com/user-attachments/assets/58cf30d1-e858-42da-81ca-2f0b66da72ae">